mosaic_builder.py

```diff
@@ -78,13 +80,16 @@ def __init__(self, iface):
         self.areaTool = areaTool(iface.mapCanvas())
         self.areaTool.canvasClicked.connect(self.selectByArea)
         self.mosaicLayer = None
-        self.currentDiscSize = 25
-        self.currentDiscArcs = False
+        self.currentDiscSize = GlobalSettings.value("mosaicBuilder/radius", None)
```

Comment:
```suggestion
        self.currentDiscSize = GlobalSettings.value("mosaicBuilder/radius", 25)
```
Doesn't the second parameter allow for a default, so instead of setting to None you can set it to 25 and then not do the check for `is None` afterwards? https://docs.qgis.org/3.40/en/docs/pyqgis_developer_cookbook/settings.html#reading-and-storing-settings
Same for the useCurves value below